src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -604,6 +644,17 @@ IParameterMixer<float> IParameterMixer<float>.CombineParameters(IList<IParameter
         }
     }
 
+    internal static class ParameterMixingCalibratedModelParameters
+    {
+        internal const string LoaderSignature = "PMixCaliPredExec";
```

Comment:
Probably don't want to duplicate these strings. The other strings should be updated to these.